### PR TITLE
check incremental backup size

### PR DESCRIPTION
**Pull request by pondrejk**
Related to issue #4638
and bug https://bugzilla.redhat.com/show_bug.cgi?id=1445989

Interestingly, tests pass, so this check probably isn't bullet-proof, in this case the second backup is still smaller than the first one:

```py.test tests/foreman/sys/test_hot_backup.py -k incremental
========================================== test session starts ==========================================
platform linux2 -- Python 2.7.11, pytest-2.9.2, py-1.4.33, pluggy-0.3.1
rootdir: /home/pondrejk/Documents/robottelo, inifile: 
plugins: xdist-1.15.0, services-1.1.14, html-1.10.0, cov-2.3.1
collected 14 items 
2017-05-05 14:32:08 - conftest - DEBUG - Found WONTFIX in decorated tests ['1110476', '1269196', '1378009', '1245334', '1221971', '1217635', '1226425', '1156555', '1199150', '1204686', '1267224', '1103157', '1230902', '1214312', '1079482']

2017-05-05 14:32:08 - conftest - DEBUG - Collected 14 test cases


tests/foreman/sys/test_hot_backup.py ..s.

================================ 10 tests deselected by '-kincremental' =================================
========================= 3 passed, 1 skipped, 10 deselected in 302.35 seconds ==========================

```

### CODE REVIEW

File: tests/foreman/sys/test_hot_backup.py [hunk 1]
```diff
@@ -46,6 +46,17 @@ def tmp_directory_cleanup(connection, *args):
         connection.run('rm -rf /tmp/{0}'.format(name))
 
 
+def directory_size_compare(connection, full_dir, inc_dir):
```
Comment (oshtaier) [inline tests/foreman/sys/test_hot_backup.py]: a little bit confusing naming for a method that return True or False

**Comment by oshtaier** [comment]
ACK

File: tests/foreman/sys/test_hot_backup.py (hunk 1 above)
Comment (sghai) [inline tests/foreman/sys/test_hot_backup.py]: compare_backup_size ?

Comment (oshtaier) [inline tests/foreman/sys/test_hot_backup.py]: I meant that when you say `compare` it is not obvious that it is True when first greater than second and vice versa

Comment (oshtaier) [inline tests/foreman/sys/test_hot_backup.py]: but I think that proper docstring will suffice our needs

**Pull request merged**